packages/shorebird_cli/lib/src/archive_analysis/linux_bundle_differ.dart

```diff
@@ -8,8 +8,13 @@ class LinuxBundleDiffer extends ArchiveDiffer {
   /// {@macro linux_bundle_differ}
   const LinuxBundleDiffer();
 
+  bool _isDirectoryPath(String path) {
+    return path.endsWith('/');
+  }
+
   @override
   bool isAssetFilePath(String filePath) =>
+      !_isDirectoryPath(filePath) &&
```

Comment:
I guess this is for zip inspection? (e.g. you're not looking at real paths on disk.

Reply:
Yes, correct